src/modules/CartPage/CartPage.module.scss

```diff
@@ -107,12 +107,22 @@
   color: $color__white;
   @include buttons-typography;
   background-color: $color__accent;
+  cursor: pointer;
+  transition: all 0.3s ease;
+
+  &:hover {
+      box-shadow: 0px 3px 13px 0px rgba(23, 32, 49, 0.40);
+    }
 
   &__DARK {
     border-radius: 0;
     color: $color__dark-theme__white;
     background-color: $color__dark-theme__accent;
   }
+
+  &__DARK:hover {
```

Comment:
you can just  write inside &__DARK &:hover and it will work nicely

Reply:
I'm not sure what you mean. Could you please provide an example for clarification?

Reply:
```suggestion
  &__DARK {
    border-radius: 0;
    color: $color__dark-theme__white;
    background-color: $color__dark-theme__accent;
    
    &:hover {
    
    }
  }
```

Reply:
One moment)